packages/discovery-react-components/src/components/StructuredQuery/StructuredQuery.tsx

```diff
@@ -19,41 +20,49 @@ export interface StructuredQueryProps {
 
 const StructuredQuery: FC<StructuredQueryProps> = ({ messages = defaultMessages }) => {
   const mergedMessages = { ...defaultMessages, ...messages };
-  const [ruleRows, setRuleRows] = useState({ rows: [{ id: 0 }] });
-  const showAddRuleRowButton = ruleRows.rows.length < MAX_NUM_SIBLING_RULE_ROWS;
+  const [structuredQuerySelection, setStructuredQuerySelection] = useState<
```

Comment:
quick question - why is it `structuredQuerySelection` vs just `structuredQuery`? just curious what the `selection` part is for

Reply:
My thinking was that it includes (and will include more) of the selections that people are making (right now, adding the rule groups, but in the future, to the operators, etc.) when building their queries. I could also be fine with just `structuredQuery`.